tensorflow/compiler/tf2tensorrt/trt_convert_api.h

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 #include <string>
 #include <vector>
 
+#include "tensorflow/cc/saved_model/loader.h"
```

Comment:
I need to remove this line, and instead just add a forward declaration
struct SavedModelBundle;